### PR TITLE
RELATED: WEB-725 Redesign the navigation experience

### PR DESCRIPTION
**Pull request by czechian**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/208/ : SUCCESS in 3m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/70/ : SUCCESS in 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/209/ : SUCCESS in 1m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/71/ : SUCCESS in 4s

File: docs/live_examples.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: _
```
Comment (no23reason) [inline docs/live_examples.md]: Detail: We should specify some meaningful title here, the big underscore looks strange.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/211/ : SUCCESS in 1m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/64/ : SUCCESS in 25s

**Pull request merged**